product_variant_configurator/models/product_attribute_value.py

```diff
@@ -7,19 +7,20 @@
 class ProductAttributeValue(models.Model):
     _inherit = "product.attribute.value"
 
-    @api.model
-    def create(self, vals):
+    @api.model_create_multi
+    def create(self, vals_list):
         """Link created attribute value to the associated template if proceed.
 
         This happens when quick-creating values from the product configurator.
         """
-        attr_value = super(ProductAttributeValue, self).create(vals)
+        attr_value = super(ProductAttributeValue, self).create(vals_list)
```

Comment:
Be consistent in the variable names:

```suggestion
        attr_values = super(ProductAttributeValue, self).create(vals_list)
```